LayoutTests/imported/w3c/web-platform-tests/svg/idlharness.window.js

```diff
@@ -171,40 +154,24 @@ idl_test(
       SVGPolygonElement: ['objects.polygon'],
       SVGTextElement: ['objects.text'],
       SVGTSpanElement: ['objects.tspan'],
-      SVGTRefElement: ['objects.tref'],
       SVGTextPathElement: ['objects.textPath'],
-      SVGAltGlyphElement: ['objects.altGlyph'],
-      SVGAltGlyphDefElement: ['objects.altGlyphDef'],
-      SVGAltGlyphItemElement: ['objects.altGlyphItem'],
-      SVGGlyphRefElement: ['objects.glyphRef'],
       SVGMarkerElement: ['objects.marker'],
-      SVGColorProfileElement: ['objects.colorProfile'],
       SVGLinearGradientElement: ['objects.linearGradient'],
       SVGRadialGradientElement: ['objects.radialGradient'],
       SVGMeshGradientElement: ['objects.meshGradient'],
       SVGStopElement: ['objects.stop'],
       SVGPatternElement: ['objects.pattern'],
       SVGClipPathElement: ['objects.clipPath'],
       SVGMaskElement: ['objects.mask'],
-      SVGCursorElement: ['objects.cursor'],
       SVGAElement: ['objects.a'],
       SVGViewElement: ['objects.view'],
       SVGScriptElement: ['objects.script'],
       SVGAnimateElement: ['objects.animate'],
+      SVGDiscardElement: ['objects.discard'],
```

Comment:
Do we have a bug for adding the SVG `<discard>` Element? If we do not have one, please file a new bug.

Reply:
I can file but Blink had the support but removed it - https://groups.google.com/a/chromium.org/g/blink-dev/c/jUJG_CFqEvE/m/bc5_tXo8BAAJ and Gecko only had `Intent to Prototype` - https://groups.google.com/a/mozilla.org/g/dev-platform/c/U4nufgUTdAk

Reply:
Then I think we should remove this line for now. This file has been out of sync for sometime. And I think it is okay to it have out of sync for the `<discard>` object since the plan for it is unclear.

I think removing this line will remove many failures from the expected file.